src/actionCreator.ts

```diff
@@ -3,7 +3,7 @@ import ActionCreator from './interfaces/ActionCreator';
 import { dispatch } from './dispatcher';
 import createActionId from './createActionId';
 
-export function actionCreator<
+export function actionCreatorWithoutDispatch<
```

Comment:
Not to rain on the parade here too much - but rarely do we see any API in the world where the identifier indicates what it DOESN'T do. It then isn't very clear what it is actually doing. I would have like something more like:

actionMessageCreator()